README.md

```diff
@@ -22,9 +22,62 @@ Once you join the The [Presto Foundation Slack Workspace](https://join.slack.com
 
-https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md
+The current members of the Presto TSC are:
+
+| Name                      | Github                                              | Affiliation | Term begins  | Term ends    |
+| ------------------------- | --------------------------------------------------- | ------------|--------------|--------------|
+| Nezih Yigitbasi (chair)   |[@nezihyigitbasi](https://github.com/nezihyigitbasi) | Facebook    | 1 Jan, 2020  | TBD          |
+| Masha Basmanova           |[mbasmanova](https://github.com/mbasmanova)          | Facebook    | 1 Jan, 2020  | TBD          |
+| Zhenxiao Luo              |[zhenxiao](https://github.com/zhenxiao)              | Twitter     | 1 Jan, 2020  | TBD          |
+| Andrii Rosa               |[arhimondr](https://github.com/arhimondr)            | Facebook    | 1 Jan, 2020  | TBD          |
+| Rebecca Schlussel         |[rschlussel](https://.github.com/rschlussel)         | Facebook    | 1 Jan, 2020  | TBD          |
+| James Sun                 |[highker](https://github.com/highker)                | Facebook    | 1 Jan, 2020  | TBD          |
+| Wenlei Xie                |[wenleix](https://github.com/wenleix)                | Facebook    | 1 Jan, 2020  | TBD          |
+| Rongrong Zhong            |[rongrong](https://github.com/rongrong)              | Facebook    | 1 Jan, 2020  | TBD          |
+| Shixuan Fan               |[shixuan-fan](https://github.com/shixuan-fan)        | Facebook    | 1 Jan, 2020  | TBD          |
+| Jiexi Lin                 |[jessesleeping](https://github.com/jessesleeping)    | Facebook    | 1 Jan, 2020  | TBD          |
+| Venki Korukanti           |[vkorukanti](https://github.com/vkorukanti)          | Uber        | 14 Feb, 2020  | TBD         |
+
+Per the [Technical Charter](CHARTER.md), when there are less than six related companies represented on the TSC, the employees of each company may only submit one vote on TSC decisions where voting is required.
+
+### Term of TSC voting members
+
+The Presto TSC seeks to balance the energy and enthusiasm of new members with the stability provided by long-term members.  As such, TSC voting members will serve three-year terms, after which they will be eligible for re-election.  There is no limit to the number of times an individual may run for an open seat.
+
+To provide balance, the terms of TSC members will be adjusted so that roughly 1/3 of seats are elected each year.  If the classes become unbalanced, then the TSC shall designate the appropriate number of seats as one-year terms at the first opportunity to fix the imbalance.
+
+For simplicity, membership terms will be aligned to the calendar year, and will run from January 1st to December 31st of the following year. Initially, the TSC shall randomly designate 1/3 of the seats to be one-year terms, 1/3 of the seats to be two-year terms, and the remainder shall be three-year terms.
```

Comment:
This randomization idea complicates the process I think. Curious what other TSC members think?

Reply:
Another option would be to ask for volunteers to serve 1 year and 2 year terms.  To be honest, the method itself isn't all that important, just that the terms are staggered in some way or another.

Reply:
Randomization seems as good a way as any if there aren't enough volunteers.  It's a one time thing just to get the term limits started, so it doesn't seem too complicated.

Reply:
how about we ask for volunteers to serve 1 year and 2 years first. If not get enough volunteers, do randomization or some other mathematical way? What do you think? @rschlussel @nezihyigitbasi @brianwarner